examples/opt_125m/user_script.py

```diff
@@ -0,0 +1,242 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+from itertools import chain
+from typing import List, Tuple
+
+import numpy as np
+import torch
+from transformers import AutoConfig
+
+from olive.constants import Framework
+
+# pylint: disable=broad-exception-raised, redefined-outer-name
+
+model_id = "facebook/opt-125m"
+config = AutoConfig.from_pretrained(model_id, trust_remote_code=True)
```

Comment:
do we need add readme? and update the example link in docs?

Reply:
Added readme. And will update the example link once this PR is merge.